tensorflow_datasets/scripts/benchmark_dataset.py

```diff
@@ -0,0 +1,48 @@
+r"""Benchmark dataset.
+
+python -m tensorflow_datasets.scripts.benchmark_dataset \
+  --dataset dataset_name
+
+"""
+import time
+
+from absl import app
+from absl import flags
+
+import tensorflow_datasets as tfds
+
+# DEFAULT_DATA_DIR = os.path.expanduser(os.path.join("~", "tensorflow_datasets"))
+
+EPOCHS = 1
```

Comment:
Rather than epoch, it's best to have `num_examples`

Reply:
Or if for you this correspond to the number of trial to average the performance across multiple trials, maybe rename this `NUM_EXECUTIONS` or `NUM_REPETITIONS` or `NUM_TRIALS`.